synthtool/gcp/templates/node_library/README.md

```diff
@@ -0,0 +1,117 @@
+{% endif %}
+
+The [{{ metadata['repo']['name_pretty'] }} {{ metadata['repo']['language']|language_pretty }} Client API Reference][client-docs] documentation
+also contains samples.
+
+## Versioning
+
+This library follows [Semantic Versioning](http://semver.org/).
+
+{% if metadata['repo']['release_level'] == 'ga' %}
+This library is considered to be **General Availability (GA)**. This means it
+is stable; the code surface will not change in backwards-incompatible ways
+unless absolutely necessary (e.g. because of critical security issues) or with
+an extensive deprecation period. Issues and requests against **GA** libraries
+are addressed with the highest priority.
+{% endif %}
+{% if metadata['repo']['release_level'] == 'beta' %}
+This library is considered to be in **beta**. This means it is expected to be
+mostly stable while we work toward a general availability release; however,
+complete stability is not guaranteed. We will address issues and requests
+against beta libraries with a high priority.
+{% endif %}
+{% if metadata['repo']['release_level'] == 'alpha' %}
+This library is considered to be in **alpha**. This means it is still a
+work-in-progress and under active development. Any release is subject to
+backwards-incompatible changes at any time.
+{% endif %}
+{% if metadata['release_level'] == 'deprecated' %}
+This library is **deprecated**. This means that it is no longer being
+actively maintained and the only updates the library will receive will
+be for critical security issues. {% if metadata['deprecated'] %}{{ metadata['deprecated'] }}{% endif %}
+{% endif %}
+
+More Information: [Google Cloud Platform Launch Stages][launch_stages]
+
+[launch_stages]: https://cloud.google.com/terms/launch-stages
+
+## Contributing
+
+Contributions welcome! See the [Contributing Guide](https://github.com/{{ metadata['repo']['repo'] }}/blob/master/CONTRIBUTING.md).
+
+## License
+
+Apache Version 2.0
+
+See [LICENSE](https://github.com/{{ metadata['repo']['repo'] }}/blob/master/LICENSE)
+
+## What's Next
+
+* [{{ metadata['repo']['name_pretty'] }} Documentation][product-docs]
+* [{{ metadata['repo']['name_pretty'] }} {{ metadata['repo']['language']|language_pretty }} Client API Reference][client-docs]
+* [github.com/{{ metadata['repo']['repo'] }}](https://github.com/{{ metadata['repo']['repo'] }})
+
+Read more about the client libraries for Cloud APIs, including the older
+Google APIs Client Libraries, in [Client Libraries Explained][explained].
+
+[explained]: https://cloud.google.com/apis/docs/client-libraries-explained
+
+[client-docs]: {{ metadata['repo']['client_documentation'] }}
+[product-docs]: {{ metadata['repo']['product_documentation'] }}
+[shell_img]: https://gstatic.com/cloudssh/images/open-btn.png
+[projects]: https://console.cloud.google.com/project
+[billing]: https://support.google.com/cloud/answer/6293499#enable-billing
+[enable_api]: https://console.cloud.google.com/flows/enableapi?apiid={{ metadata['repo']['api_id'] }}
```

Comment:
my goodness, I can't believe I spent 20 minutes today reading this as `app_id`; sorry for inflicting my 🧠 :zzz: on you @jmdobry/@busunkim96.